puffin_http/Cargo.toml

```diff
@@ -25,3 +25,5 @@ puffin = { version = "0.19.0", path = "../puffin", features = [
 
 [dev-dependencies]
 simple_logger = "4.2"
+paste = "1.0.15"
```

Comment:
These deps are here so that the example code can be tested in the docs for `Server::new_custom()`. 

We could modify the examples if needed to remove this, but it would be a decent amount of work